Upstream-Status: Inappropriate [embedded specific]

8/27/2010 - created by Qing He <qing.he@intel.com>
2013-12-27 - rebased for stlinux version by Stefan Seyfried <seife@tuxbox-git.slipkontur.de>

Index: b/src/Makefile
===================================================================
--- a/src/Makefile
+++ b/src/Makefile
@@ -4,7 +4,6 @@ includedir=$(prefix)/include
 libdir=$(prefix)/lib
 libdevdir=$(prefix)/lib
 
-ARCH := $(shell uname -m | sed -e s/i.86/i386/)
 BASE_CFLAGS := -nostdlib -nostartfiles -fomit-frame-pointer -fPIC $(CFLAGS)
 BASE_CPPFLAGS := -I. $(CPPFLAGS)
 CFLAGS := -Wall -g -O2
@@ -48,8 +47,8 @@ $(libaio_objs) $(libaio_sobjs): libaio.h
 
 libaio.a: $(libaio_objs)
 	rm -f libaio.a
-	ar r libaio.a $^
-	ranlib libaio.a
+	$(AR) r libaio.a $^
+	$(RANLIB) libaio.a
 
 $(libname): $(libaio_sobjs) libaio.map
 	$(CC) $(SO_CFLAGS) -Wl,--version-script=libaio.map -Wl,-soname=$(soname) -o $@ $(libaio_sobjs) $(LINK_FLAGS)
